tests/functional/base.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+import time
+import unittest
+
+import requests
+
+from custodia.client import HTTPUnixAdapter
+from custodia.server.args import parse_args
+from custodia.server.config import parse_config
+
+
+class CustodiaServerBase(unittest.TestCase):
+
+    request_headers = {'REMOTE_USER': 'me'}
+
+    def __init__(self, configfile=None, configstr=None):
```

Comment:
In my opinion it is better to use `setUpClass()` instead of `__init__()`.
( see https://docs.python.org/3/library/unittest.html#unittest.TestCase )